oozie-to-airflow/examples/spark/job.properties

```diff
@@ -36,4 +36,4 @@ master=local[*]
 queueName=default
 examplesRoot=examples
 oozie.use.system.libpath=true
-oozie.wf.application.path=${nameNode}/user/${user.name}/${examplesRoot}/apps/spark
+oozie.wf.application.path=${nameNode}/user/${user.name}/spark
```

Comment:
This path is not good for automated execution. It should follow the same scheme as all the others examples.

Reply:
Thanks. I updated a PR.